docs/extensibility/available-slot-fills.md

```diff
@@ -0,0 +1,37 @@
+# Available Slots
+
+To see how to use a slotFill, [check this document](./slot-fills.md).
+
+This is a list of available slots that you can use. If you want to add a new slotFill, [check this document](../../pacakges/checkout/slot/README.md).
+
+## ExperimentalOrderMeta
```

Comment:
It might be worth providing a link to an explanation about feature gating in the blocks plugin and the convention for how slots and fills are named right now.